metricbeat/tests/system/test_system.py

```diff
@@ -3,11 +3,14 @@
 import unittest
 import metricbeat
 
-SYSTEM_CPU_FIELDS = ["idle", "iowait", "irq", "load", "nice", "softirq",
-                     "steal", "system", "system_p", "user", "user_p"]
+SYSTEM_CPU_FIELDS = ["idle_p", "iowait_p", "irq_p", "load", "nice_p",
+                     "softirq_p", "steal_p", "system_p", "user_p"]
 
-SYSTEM_CORE = ["core", "idle", "iowait", "irq", "nice", "softirq",
-                "steal", "system", "system_p", "user", "user_p"]
+SYSTEM_CPU_ALL_FIELDS = ["idle_p", "idle", "iowait_p", "iowait", "irq_p", "irq", "load", "nice_p", "nice",
```

Comment:
@tsg agree on both (suggestion and future PR). This is even better the moving all ticks into a namespace "ticks". Also if you want to filter out `idle` in the example above, you just define `system.cpu.idle.*` and all idle information is removed.